### ISSUE TITLE
Doorbell text sensor?

### ISSUE THREAD

**Issue by irgusite** [briis/unifiprotect#263]
Would it be possible to have a sensor with the actual text of the doorbell? 
Would be nice to have it on the dashboard :)

Thank you very much for this awesome integration

**Comment by briis** [comment]
I will look in to this.

**Comment by briis** [comment]
FYI, I will postpone this a bit, and then use the new `select` entity, to give you a list of the pre-defined texts. That way you can both display it on your dashboard, and also set it directly from there.

**Comment by edwardmp** [comment]
Opening this again. I'm running v0.10.1, and although I can change the text to the preset options, there is currently no way to obtain the text that is currently on the display (the display shows a dynamic text that is updated by HA).

**Comment by AngellusMortis** [comment]
Upgrade 0.11-beta. The Doorbell select has been rewritten to use the Custom Message values from UniFi Protect. So, add your custom messages via the new `unifiprotect.add_doorbell_text` service or via the Android/iOS mobile app and they will appear as select options.

If you have a message that is not in your Custom Messages list, it will still show up blank, but having a Sensor entity and the Select entity will duplicate data in HA (which is really bad).

**Comment by edwardmp** [comment]
Do I understand correctly that if I have a dynamic message (e.g. I show the current temperature on the display) there is no way to get the current value?

**Comment by AngellusMortis** [comment]
Correct. There is no way. If you are setting a dynamic value from Home Assistant, you should be setting the value from another entity so you can keep the two in sync and use that entity to monitor that is current on there.
